test/fixtures/helpers/moar-helpers/random.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
```

Comment:
Note that I threw a bunch of helpers from our core-hbs-helpers module into fixtures because they were handy to use as such. We shouldn't need to worry too much about maintaining these; I just needed some warm bodies or whatever.